packages/rfw/lib/src/flutter/core_widgets.dart

```diff
@@ -64,56 +61,82 @@ import 'runtime.dart';
 ///
 /// For each, every parameter is implemented using the same name. Parameters
 /// that take structured types are represented using maps, with each named
-/// parameter of that type's default constructor represented by a key, with the
-/// following notable caveats and exceptions:
+/// parameter of that type's default constructor represented by a key.
+///
+/// For example, a [TextStyle] argument of a [Text] widget would look like this:
+///
+/// ```rfwtxt
+/// Text(
+///   text: 'hello',
```

Comment:
Text takes a ListString>:
`'Text': (BuildContext context, DataSource source) {
    String? text = source.v<String>(['text']);
    if (text == null) {
      final StringBuffer builder = StringBuffer();
      final int count = source.length(['text']);
      for (int index = 0; index < count; index += 1) {
        builder.write(source.v<String>(['text', index]) ?? '');
      }
      text = builder.toString();
    }`